src/components/Chart.js

```diff
@@ -1,5 +1,64 @@
-import React from "react";
+import React, { Component } from "react";
+import ChartComponent, { Line } from "react-chartjs-2";
 
-const Chart = () => <div>Chart Component</div>;
+let values = [
+  {
+    date: "some_date_1",
+    value: 1
+  },
+  {
+    date: "some_date_2",
+    value: 2
+  },
+  {
+    date: "some_date_3",
+    value: 3
+  }
+];
+
+class Chart extends Component {
+  constructor(props) {
```

Comment:
This Chart should be used like this:
<Chart values={variableContainingDatesAndValues}>

These values^ will be then passed as props.values.
Please read more about props in react and improve this code.